browser.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env browser */
+import {createStringGenerator, createAsyncStringGenerator} from './core.js';
+
+const toHex = uInt8Array => uInt8Array.map(byte => byte.toString(16).padStart(2, '0')).join('');
+
+const decoder = new TextDecoder('utf8');
+const toBase64 = uInt8Array => btoa(decoder.decode(uInt8Array));
+
+// `crypto.getRandomValues` throws an error if too much entropy is requested at once. (https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues#exceptions)
+const maxEntropy = 65536;
+
+function getRandomValues(byteLength) {
+	const generatedBytes = [];
+
+	while (byteLength > 0) {
+		const bytesToGenerate = Math.min(byteLength, maxEntropy);
+		generatedBytes.push(crypto.getRandomValues(new Uint8Array({length: bytesToGenerate})));
```

Comment:
Why two loops instead of adding to the `result` directly?

Reply:
`Uint8Array`s have a fixed size so the final size needs to be determined after all the arrays are collated. Otherwise, we'd need to recreate the array every loop.